src/Documentation/CookbookGenerator.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Documentation;
+
+/**
+ * @internal
+ */
+final class CookbookGenerator
+{
+    private const TEMPLATES_PATH = __DIR__.'/../../dev-tools/cookbook/';
+
+    public function generateDocumentation(): string
+    {
+        $fixerContent = file(self::TEMPLATES_PATH.'RemoveCommentsFixer.php');
+        $testContent = file(self::TEMPLATES_PATH.'RemoveCommentsFixerTest.php');
+
+        return strtr(
```

Comment:
this is not readable at all and I doubt it makes sense to maintain it

Reply:
it is terrible, but isn't still better to have it automated (see how much cookbook changed) than not?